configuration/concepts/commons.csv

```diff
@@ -2,4 +2,10 @@ Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Fully sp
 67218dc9-aee7-40d1-ba78-d29dd2731333,,CIEL:1362,One plus,+,Uno más,+,,Misc,N/A,,,,,,,,,,,
 55800eb1-4e0f-4a75-a12b-aa1e10c64e81,,CIEL:1363,Two plus,++,Dos más,++,,Misc,N/A,,,,,,,,,,,
 17327c32-0a60-4f76-85b5-24379fc72ad5,,CIEL:1364,Three plus,+++,Tres más,+++,,Misc,N/A,,,,,,,,,,,
-f0853fb9-5541-4e13-b79a-e3b438a629cb,,CIEL:1365,Four plus,++++,Cuatro más,++++,,Misc,N/A,,,,,,,,,,,
\ No newline at end of file
+f0853fb9-5541-4e13-b79a-e3b438a629cb,,CIEL:1365,Four plus,++++,Cuatro más,++++,,Misc,N/A,,,,,,,,,,,
+286aa7d3-d7c2-4252-bc13-c50afc032bbf,,,Right,Right,Derecha,Derecha,,Misc,N/A,,,,,,,,,,,
+288a030a-2fed-43d1-9c24-32e0fb02f73d,,,Left,Left,Izquierda,Izquierda,,Misc,N/A,,,,,,,,,,,
+1712fd14-96a6-469d-877c-c691040338ee,,,Anatomical Position,Anatomical Position,Posicion anatomica,Posicion anatomica,,Misc,Coded,,Right; Left,,,,,,,,,
+2db269b5-ca96-4c3f-998a-88f61361eb2b,,,Body Side,Body Side,Lado del cuerpo,Lado del cuerpo,,Misc,Coded,,Right; Left,,,,,,,,,
```

Comment:
Let's add a concept called 'Anatomical position' FSN and SSN and 'Body side' coded with answer 'Right/Left'. Add this to the csv called commons.csv
From this comment before....this should be one concept.

Reply:
Done